Resources/Prototypes/_NF/Recipes/Cooking/meal_recipes.yml

```diff
@@ -0,0 +1,1742 @@
+  result: LeavesCannabisDried
+  time: 10
+  solids:
+    LeavesCannabis: 1
+
+- type: microwaveMealRecipe
+  id: RecipeAloeCream
+  name: aloe cream recipe
+  result: AloeCream
+  time: 10
+  solids:
+    FoodAloe: 1
+
+- type: microwaveMealRecipe
+  id: RecipeMedicatedSuture
+  name: medicated suture recipe
+  result: MedicatedSuture
+  time: 10
+  solids: 
+    FoodPoppy: 1
+    Brutepack: 1
+    MaterialCloth1: 1
+  reagents:
+    TranexamicAcid: 50
+    Cryptobiolin: 50
+
+- type: microwaveMealRecipe
+  id: RecipeRegenerativeMesh
+  name: regenerative mesh recipe
+  result: RegenerativeMesh
+  time: 10
+  solids:
+    FoodAloe: 1
+    Ointment: 1
+    MaterialCloth1: 1
+  reagents:
+    Sigynate: 50
+    Dermaline: 50 
+
+# Suppermatter
+- type: microwaveMealRecipe
+  id: RecipeSuppermatter
+  name: suppermatter recipe
+  result: FoodCakeSuppermatter
+  time: 30
+  solids:
+    FoodCakeBatter: 2
+  reagents:
+    Sugar: 30
+    Nitrogen: 10
+    Plasma: 10
+
+# Kebabs
+- type: microwaveMealRecipe
+  id: RecipeMeatKebab
+  name: meat kebab recipe
+  result: FoodMeatKebab
+  time: 5
+  solids:
+    FoodMeatCutlet: 3
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeHawaiianKebab
+  name: Hawaiian kebab recipe
+  result: FoodMeatHawaiianKebab
+  time: 5
+  solids:
+    FoodChili: 1
+    FoodMeatCutlet: 1
+    FoodPineappleSlice: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeFiestaKebab
+  name: fiesta kebab recipe
+  result: FoodMeatFiestaKebab
+  time: 5
+  solids:
+    FoodChili: 1
+    FoodCorn: 1
+    FoodMeatCutlet: 1
+    FoodTomato: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeRatKebab
+  name: rat kebab recipe
+  result: FoodMeatRatKebab
+  time: 10
+  solids:
+    FoodMeatRat: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeDoubleRatKebab
+  name: double rat kebab recipe
+  result: FoodMeatRatdoubleKebab
+  time: 20
+  solids:
+    FoodMeatRat: 2
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeHumanKebab
+  name: human kebab recipe
+  result: FoodMeatHumanKebab
+  time: 15
+  solids:
+    TorsoHuman: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeLizardKebab
+  name: lizard kebab recipe
+  result: FoodMeatLizardtailKebab
+  time: 15
+  solids:
+    FoodMeatLizard: 1
+    FoodKebabSkewer: 1
+
+- type: microwaveMealRecipe
+  id: RecipeFoodMealTaco
+  name: taco recipe
+  result: FoodMealTaco
+  time: 10
+  solids:
+    FoodDoughSlice: 1
+    FoodTomato: 1
+    FoodOnionSlice: 2
+    FoodCheeseSlice: 1
+    FoodMeatCutlet: 1
+
+- type: microwaveMealRecipe
+  id: RecipeFoodBakedChevreChaud
+  name: chevre chaud recipe
+  result: FoodBakedChevreChaud
+  time: 5
+  solids:
+    FoodChevreSlice: 1
+    FoodBreadBaguetteSlice: 1
+
+- type: microwaveMealRecipe
+  id: RecipeCookedSpaceshroom
+  name: cooked spaceshroom recipe
+  result: FoodSpaceshroomCooked
+  time: 5
+  solids:
+    FoodSpaceshroom: 1
+
```

Comment:
looks like you forgot to delete everything above here in this file when you did your copy + paste